src/Analytics/Adapter/GoogleAnalytics.php

```diff
@@ -0,0 +1,122 @@
+<?php
+
+/**
+ * Utopia PHP Framework
+ *
+ * @package Analytics
+ * @subpackage Tests
+ *
+ * @link https://github.com/utopia-php/framework
+ * @author Torsten Dittmann <torsten@appwrite.io>
+ * @version 1.0 RC1
+ * @license The MIT License (MIT) <http://www.opensource.org/licenses/mit-license.php>
+ */
+
+namespace Utopia\Analytics\Adapter;
+
+use Utopia\Analytics\Adapter;
+use Utopia\Analytics\Event;
+
+class GoogleAnalytics extends Adapter
+{
+    /**
+     *  Endpoint for Google Analytics
+     *  @var string
+     */
+    public string $endpoint = 'https://www.google-analytics.com/collect';
+
+    /**
+     * Tracking ID for Google Analytics
+     * @var string
+     */
+    private string $tid;
+
+    /**
+     * A unique identifer for Google Analytics
+     * @var string
+     */
+    private string $cid;
+
+    /**
+     * Gets the name of the adapter.
+     * 
+     * @return string
+     */
+    public function getName(): string
+    {
+        return 'GoogleAnalytics';
+    }
+
+    /**
+     * @param string $tid 
+     * The tracking ID / web property ID. The format is UA-XXXX-Y. All collected data is associated by this ID.
+     * 
+     * @param string $cid
+     * This pseudonymously identifies a particular user, device, or browser instance.
+     * 
+     * @return GoogleAnalytics
+     */
+    public function __construct(string $tid, string $cid)
+    {
+        $this->tid = $tid;
+        $this->cid = $cid;
+    }
+
+    /**
+     * Creates an Event on the remote analytics platform.
+     * 
+     * @param Event $event
+     * @return bool
+     */
+    public function createEvent(Event $event): bool 
+    {
+        if (!$this->enabled) {
+            return false;
+        }
+
+        $query = [
+            'ea' => $event->getProps('action'),
+            't' => $event->getType()
+        ];
```

Comment:
```suggestion
        $query = [
            'ec' => ,
            'ea' => $event->getProps('action'),
            'el' => $event->getName(),
            'ev' => $event->getValue(),
            'dh' => ,
            'dp' => ,
            'dt' => ,
            't' => $event->getType()
        ];
        
        $query = array_filter($query, fn($value) => !is_null($value) && $value !== '');
```
We create the complete query array, and finally filter any null / empty values. 
Do you see a problem with doing something like this ? It might be much easier and we could avoid all the ifs ?

Reply:
Please fill the remaining values too :)